.github/workflows/stale.yaml

```
@@ -35,16 +35,16 @@ jobs:
     - uses: actions/stale@v7
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: "This issue is stale because it has been open for 90 days with no activity. Remove stale label or comment or this will be closed in 60 days."
-        stale-pr-message: "This PR is stale because it has been open for 90 days with no activity. Remove stale label or comment or this will be closed in 60 days."
+        stale-issue-message: "This issue is stale because it has been open for 90 days with no activity. Remove stale label or comment or this will be closed in 90 days."
+        stale-pr-message: "This PR is stale because it has been open for 90 days with no activity. Remove stale label or comment or this will be closed in 90 days."
 
         days-before-stale: 90
-        days-before-close: 60
+        days-before-close: 90
```

I'd say let's do 120 for now.
```suggestion
        days-before-close: 120
```